pkg/features/list.go

```diff
@@ -116,6 +116,9 @@ var (
 
 	// Flattens CVE Data Model for improved accuracy and performance
 	FlattenCVEData = registerFeature("Uses a flattened CVE Data Model improved accuracy and performance", "ROX_FLATTEN_CVE_DATA")
+
+	// Adds the ability to generate vulnerability reports based on filter views
+	VulnerabilityAdHocReports = registerFeature("Adds the ability to generate vulnerability reports based on filter views", "ROX_VULNERABILITY_AD_HOC_REPORTS",enabled)
```

Comment:
Why is this new feature enabled by default? Is it production ready?

Reply:
We can set it to false if that is the preference. It is expected to be ready in this release.

Reply:
Yes, please set it to enabled only when the feature is production ready.